themes/default/layouts/docs/list.html

```diff
@@ -28,7 +45,7 @@ <h1>{{ .Title }}</h1>
                 </div>
             </div>
 
-            <div class="lg:w-3/12 pr-8 mb-2 pt-8 lg:order-first lg:border-none lg:pt-0 lg:mt-0">
+            <!-- <div class="hidden lg:block lg:w-3/12 pr-8 mb-2 pt-8 lg:order-first lg:border-none lg:pt-0 lg:mt-0">
```

Comment:
Should we not just remove this?

Reply:
oh 💯, i was looking at so much code that I missed not deleting this.  will get it + fix this up tomorrow, including dropped the two svgs @susanev noticed that are already in master.  i probably shoulda just taken a coffee break instead of tackling a new pr 😬